src/Orleans/Logging/TraceLogger.cs

```diff
@@ -266,19 +266,47 @@ public static void Initialize(ITraceConfiguration config, bool configChange = fa
                     {
                         LogConsumers.Add(new LogWriterToConsole());
                     }
+
+                    bool containsConsoleTelemetryConsumer = false;
+                    foreach (var tc in TelemetryConsumers)
+                    {
+                        if (tc.GetType() != typeof(ConsoleTelemetryConsumer)) continue;
+
+                        containsConsoleTelemetryConsumer = true;
+                        break;
+                    }
+
+                    if (!containsConsoleTelemetryConsumer)
+                    {
+                        TelemetryConsumers.Add(new ConsoleTelemetryConsumer());
+                    }
                 }
                 if (!string.IsNullOrEmpty(config.TraceFileName))
                 {
                     try
                     {
-                        logOutputFile = new FileInfo(config.TraceFileName);
-                        var l = new LogWriterToFile(logOutputFile);
+                        var l = new LogWriterToFile();
```

Comment:
I'm confused here. We'll use both a `LogWriterToFile` and a `FileTelemetryConsumer` in parallel?